src/TimePicker/TimeDisplay.js

```diff
@@ -70,7 +70,7 @@ class TimeDisplay extends Component {
         borderTopLeftRadius: 2,
         borderTopRightRadius: 2,
         backgroundColor: timePicker.headerColor,
-        color: 'white',
+        color: timePicker.textColor,
```

Comment:
That's a breaking change. We can't merge like that.
```jsx
timePicker.textColor = palette.accent3Color
```

I think that we should change that line with `palette.textColor`
https://github.com/callemall/material-ui/blob/master/src/styles/getMuiTheme.js#L296